src/preferences/dialog/dlgprefcontrolsdlg.ui

```diff
@@ -413,6 +413,33 @@
           </attribute>
          </widget>
         </item>
+        <item row="3" column="0">
+         <widget class="QLabel" name="labelKeyunlockMode">
+          <property name="text">
+           <string>Keyunlock mode</string>
+          </property>
+         </widget>
+        </item>
+        <item row="3" column="1">
+         <widget class="QRadioButton" name="radioButtonCalculateNewKey">
+          <property name="text">
+           <string>Adapt key to current tempo</string>
```

Comment:
It looks like you do not like the term "linear pitch" ... Normally I would pick here "Reset to linear pitch"
"Adapt key to current tempo" implies a of magic to me. 
How about just "Reset key" and "Keep key"? 

Other idea: "Reset to turntable pitch"